tools/amd_build/pyHIPIFY/hipify_python.py

```diff
@@ -45,6 +45,33 @@
 CAFFE2_TEMPLATE_MAP = {}
 
 
+class Error(Exception):
+    """Base class for exceptions in this module."""
+
+    def __init__(self, expression, message, desc):
```

Comment:
The only place uses InputError is only passing one argument:
https://github.com/pytorch/pytorch/blob/d05b4e31d1a41bf5c908adb2d3a0d4d6ba0118fc/tools/amd_build/pyHIPIFY/hipify_python.py#L422